beta/nncf/tensorflow/api/compression.py

```diff
@@ -81,6 +81,10 @@ class TFCompressionAlgorithmBuilder(CompressionAlgorithmBuilder):
     order to enable algorithm-specific compression during fine-tuning.
     """
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._op_names = set()
```

Comment:
@daniil-lyakhov I suggest not changing the abstract class. Let's remove op_names from the abstract class and use the relevant attributes in the inherited classes.

Reply:
Please check